packages/blanket/src/styled.ts

```diff
@@ -9,4 +9,5 @@ export const Root = styled.div<IBlanketProps>`
 
   opacity: ${({ theme, isTinted }: IBlanketProps) => (isTinted ? getBlanketThemeStylesByProperty({ theme })(['opacity']) : 0)};
   pointer-events: ${({ canClickThrough }: IBlanketProps) => (canClickThrough ? 'none' : 'initial')};
+  z-index: ${({ theme, maxZIndex }) => (maxZIndex ? maxZIndex + 1 : blanketTheme(theme, ['zIndex']))};
```

Comment:
I think this logic shouldn't be placed here. It is more correct put it inside component.

Reply:
Why? It is normal practice to choose different style properties by the props. You can see similar logic in two lines above. What is different between this line and two lines above?